mars_rover/spec/traveller_spec.rb

```diff
@@ -37,18 +35,20 @@
     end
 
     context "given a set of coordinates and a traversal path" do
-      it "should get the final position wrt after the entire traversal" do
+      context "traveller travels within area of travel"  do
+        it "should get the final position wrt after the entire traversal" do
 
-        loc2 = Location.new(3,3,"E")
-        path2 = 'MMRMMRMRRM'
-        traveller2 = Traveller.new(loc2, path2)
+          loc2 = Location.new(3,3,"E")
+          direction2 = Direction.new('MMRMMRMRRM')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.